src/service/PkiService.js

```diff
@@ -40,6 +40,7 @@ exports.createDFSP = async (ctx, body) => {
     dfsp_id: body.dfspId,
     name: body.name,
     monetaryZoneId: body.monetaryZoneId ? body.monetaryZoneId : undefined,
+    isProxy: body.isProxy,
```

Comment:
Do we need these details stored in PKI service?

Reply:
These are DB related functions, I am not sure why they were originally implemented here.